.github/workflows/ci-check-license.yaml

```diff
@@ -0,0 +1,38 @@
+#
+#  Copyright 2022 The KubeSphere Authors.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+name: Check License Header
+
+on:
+  push:
+  pull_request:
+  release:
+
+jobs:
+  Build:
+    name: Check License Header
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Check License Header
+        uses: apache/skywalking-eyes@main
```

Comment:
It would be better to use fixed version, like commit id, instead of main. See https://github.com/apache/skywalking-eyes#github-actions for more.

```yaml
- name: Check License Header
  uses: apache/skywalking-eyes@main      # always prefer to use a revision instead of `main`.
```

Reply:
Done.